tests/unittests/tests-sock_util/tests-sock_util.c

```diff
@@ -205,47 +194,57 @@ static void test_sock_util_str2ep__ipv4_port(void)
 static void test_sock_util_str2ep__ipv4_bracketed(void)
 {
     sock_udp_ep_t ep;
-    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep,
-                                                   TEST_STR2EP_V4_INVALID));
+    /* IPv4 addr enclosed in IPv6 Brackets */
+    const char * test_str =  "[10.0.0.1]:53";
+    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep, test_str));
 }
 
+
 static void test_sock_util_str2ep__invalid_bracket_missing(void)
 {
     sock_udp_ep_t ep;
-    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep,
-                                                   TEST_STR2EP_NETIF_INVALID3));
+    /* IPv6 addr missing closing bracket */
+    const char * test_str = "[fe80::1%53:4232";
+    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep, test_str));
 }
 
 static void test_sock_util_str2ep__invalid_ipv6(void)
 {
     sock_udp_ep_t ep;
-    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep, TEST_STR2EP_INVALID));
+    /* IPv6 addr has too many parts 9 > 8 */
+    const char * test_str = "[2001:db8:a:b:c:d:e:f:1]";
+    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep, test_str));
 }
 
 static void test_sock_util_str2ep__invalid_netif_missing(void)
 {
     sock_udp_ep_t ep;
-    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep,
-                                                   TEST_STR2EP_NETIF_INVALID));
+    /* netif part is 0 lenght/missing */
+    const char * test_str = "[fe80::1%]:752";
+    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep, test_str));
 }
 
 static void test_sock_util_str2ep__invalid_netif(void)
 {
     sock_udp_ep_t ep;
-    TEST_ASSERT_EQUAL_INT(-EINVAL, sock_udp_str2ep(&ep,
-                                                   TEST_STR2EP_NETIF_INVALID2));
+    /* netif_str longer than NETIF_STR_LEN - 1 (>4) */
```

Comment:
I am not sure if this is the intention of that test, but that is what it tests for (afaik)